float64[] pose0
float64[] levels
front_following/Sampled_Path[] paths
